src/control/destination/background/client.rs

```diff
@@ -1,16 +1,46 @@
+use std::{
+    fmt,
+    io,
+    time::{Duration, Instant},
+};
+
+use bytes::Bytes;
 use futures::{Async, Future, Poll};
+use h2;
 use http;
-use std::fmt;
-use std::io;
-use std::time::{Duration, Instant};
 use tokio::timer::Delay;
-use tower_service::Service;
-use tower_h2;
-use tower_reconnect::{Error as ReconnectError};
-
-use timeout::TimeoutError;
 
-// ===== Backoff =====
+use tower_h2::{self, BoxBody};
+use tower_service::Service;
+use tower_reconnect::{Reconnect, Error as ReconnectError};
+use conditional::Conditional;
+use dns;
+use timeout::{Timeout, TimeoutError};
+use transport::{tls, HostAndPort, LookupAddressAndConnect};
+use watch_service::Rebind;
+
+/// Type of the client service stack used to make destination requests.
+pub(super) type ClientService = AddOrigin<Backoff<LogErrors<Reconnect<
```

Comment:
If this was defined as
```rust
pub(super) struct ClientService(AddOrigin<Backoff<LogErrors<Reconnect<
        tower_h2::client::Connect<
            Timeout<LookupAddressAndConnect>,
            ::logging::ContextualExecutor<
                ::logging::Client<
                    &'static str,
                    HostAndPort
                >
            >,
            BoxBody,
        >
    >>>>)
```
instead of a type alias, then the `AddOrigin`, `Backoff`, and `LogErrors` types wouldn't have to be `pub(super)`. I'm considering making that change in a subsequent branch.